cmd/k8s-operator/operator.go

```diff
@@ -547,7 +549,8 @@ func (a *ServiceReconciler) newAuthKey(ctx context.Context, tags []string) (stri
 			},
 		},
 	}
-	key, _, err := a.tsClient.CreateKey(ctx, caps)
+	ninetyDays := 90 * 24 * time.Hour
```

Comment:
Recommend passing in zero, which will cause the `omitempty` of the keyRequest to leave ExpirySeconds unpopulated, which will inherit the server default.